backend/src/schemas/contact.schema.ts

```diff
@@ -0,0 +1,27 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document } from 'mongoose';
+
+export type ContactDocument = Contact & Document;
+
+@Schema()
+export class Contact {
+  @Prop()
+  firstName: string;
+
+  @Prop()
+  profilePictureS3Id: string;
+
+  @Prop()
+  companyName: string;
+
+  @Prop()
+  logoS3Id: string;
+
+  @Prop()
+  position: string;
```

Comment:
What do you think about renaming this to `companyPosition`?

Reply:
I think it's great! Done :heavy_check_mark: